modules/ROOT/pages/ch2-api.adoc

```diff
@@ -0,0 +1,376 @@
+= CloudHub 2.0 REST API
+
+*Last update* 02-Mar-22
+
+Added all the endpoints from https://github.com/mulesoft/rtf-management-service/blob/master/assets/raml/public/api.raml[GitHub].
+
+* *Private Spaces*
+** Get the list of private spaces in the organization.
+** Create a private space.
+** Delete the private space.
+** Get the private space. *What does this mean?*
```

Comment:
Get the details of a single Private Space (vs the list action above). This endpoint reveals more detailed information of the Private Spaces compared to the list endpoint.

Reply:
Updated